pyriemann/classification.py

```diff
@@ -141,7 +140,7 @@ def _predict_distances(self, X):
         else:
             dist = Parallel(n_jobs=self.n_jobs)(delayed(distance)(
                 X, self.covmeans_[m], self.metric_dist)
-                for m in range(n_centroids))
+                                                for m in range(n_centroids))
```

Comment:
I do agree.

Reply:
I did not even realize the format of other functions had changed.... Actually, there are also several around softmax!
It might be due to the reformat option in Pycharm. Please let me know if you find others.